docs/sources/administration/correlations/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Correlations
+weight: 900
+keywords:
+  - correlations
+  - Explore
+---
+
+# Correlations
+
+You can create interactive links for Explore visualizations to run queries related to presented data by setting up Correlations.
+
+A correlation defines how data in one [data source]({{< relref "/docs/grafana/latest/datasources/" >}}) is used to query data in another data source. Some examples:
+
+- an application name returned in a logs data source can be used to query metrics related to that application in a metrics data source, or
+- a user name returned by an SQL data source can be used to query logs related to that particular user in a logs data source
+
+[Explore]({{< relref "/docs/grafana/latest/explore/" >}}) takes user-defined correlations to display links inside the visualizations. You can click on a link to run the related query and see results in [Explore Split View]({{< relref "/docs/grafana/latest/explore/#split-and-compare" >}}).
+
+Explore visualizations that currently support showing links based on correlations:
+
+- [Logs Panel]({{< relref "./use-correlations-in-visualizations#correlations-in-logs-panel">}})
+- [Table]({{< relref "./use-correlations-in-visualizations#correlations-in-table">}})
```

Comment:
what about the graph panel? it's actually possible to get links there as well

Reply:
It's possible to add but they don't fully work as expected. The same with TraceView panel. We have issues to make them fully compatible in:

* #58441
* #59092